pkg/pxc/backup/restore.go

```diff
@@ -253,7 +407,7 @@ func S3RestoreJob(cr *api.PerconaXtraDBClusterRestore, bcp *api.PerconaXtraDBClu
 		app.GetSecretVolumes("vault-keyring-secret", cluster.PXC.VaultSecretName, true),
 	}
 	pxcUser := "xtrabackup"
-	command := []string{"recovery-s3.sh"}
+	command := []string{"recovery-cloud.sh"}
```

Comment:
@pooknull we can use the new script  only from 'v1.12.0'

Reply:
Wouldn't it be acceptable to only ship this PR with v1.12 (given this PRs milestone) anyway?

Reply:
@EraYaN  We will ship this PR with v1.12 but as you know our operator can support different versions of DB clusters https://github.com/percona/percona-xtradb-cluster-operator/blob/main/deploy/cr.yaml#L13. e.g. operator  v1.12.0 can work with 1.10.0, 1.11.0, and 1.12.0. We need to use a new script only in case of crVersion >= 1.12.0 because only backup images starting from 1.12.0 will have it.